Tiltfile

```diff
@@ -256,12 +263,12 @@ def build_node_yaml():
             if wormchain:
                 container["command"] += [
                     "--wormchainWS",
-                    "ws://guardian-validator:26657/websocket",
+                    "ws://wormchain:26657/websocket",
                     "--wormchainLCD",
-                    "http://guardian-validator:1317"
+                    "http://wormchain:1317"
```

Comment:
Let's update this so that each guardian connects to the corresponding wormchain node? For example, `guardian-0` connects to `wormchain-0` and `guardian-1` connects to `wormchain-1`

Reply:
These configs apply to the StatefulSet (all pods). To apply configs per instance we would ether need to do it with an InitContainer and ConfigMap, or at runtime like the guardian key (quicker & easier). We need to do this for the supplying wormchain keys anyway, adding unique RPC addresses will be trivial once the key-loading is in place.

I captured this in an issue https://github.com/wormhole-foundation/wormhole/issues/2246 and added it to the milestone.